server/README.md

```diff
@@ -34,7 +34,7 @@ var (
 		},
 	}
 
-	encodingConfig = simapp.MakeEncodingConfig()
+	encodingConfig = simapp.MakeTestEncodingConfig()
```

Comment:
Any idea how we should update this README file?

Reply:
It seams that the `MakeTestEncodingConfig` is needed to users who want to use `cli`.

Reply:
No it shouldn't be. Not sure why this is here.

Reply:
That example code needs to be rewritten to not use EncodingConfig

Reply:
> No it shouldn't be. Not sure why this is here.

Another point that this task is useful ;)